src/main/java/com/zufar/onlinestore/payment/controller/PaymentController.java

```diff
@@ -1,37 +1,49 @@
 package com.zufar.onlinestore.payment.controller;
```

Comment:
Why it is controller package, not endpoint package like it is everywhere in our project?